manifests/t/TheDocumentFoundation/LibreOffice/HelpPack/24.2.2.2/TheDocumentFoundation.LibreOffice.HelpPack.installer.yaml

```diff
@@ -0,0 +1,32 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: TheDocumentFoundation.LibreOffice.HelpPack
+PackageVersion: 24.2.2.2
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: msi
+Scope: machine
+UpgradeBehavior: install
+Dependencies:
+  ExternalDependencies:
+  - LibreOffice.LibreOffice
+ReleaseDate: 2024-03-25
+Installers:
+- Architecture: x86
+  InstallerUrl: https://downloadarchive.documentfoundation.org/libreoffice/old/24.2.2.2/win/x86/LibreOffice_24.2.2.2_Win_x86_helppack_en-US.msi
+  InstallerSha256: 397DBBB167681C6AFAD0C33E7869C779D3A152A2AE41CEEEA081864E4092DCF1
+  ProductCode: '{DD10966F-8F0A-4F19-BF1D-5CB05C669021}'
+  AppsAndFeaturesEntries:
+  - DisplayName: LibreOffice 24.2 Help Pack (English (United States))
+    UpgradeCode: '{40C2DB8E-E9D6-4451-BE31-DAD1343EC3DF}'
+- Architecture: x64
+  InstallerUrl: https://downloadarchive.documentfoundation.org/libreoffice/old/24.2.2.2/win/x86_64/LibreOffice_24.2.2.2_Win_x86-64_helppack_zh-TW.msi

```

Comment:
Really? The description says "LibreOffice 24.2 Help Pack (English (United States))" and the URL says Taiwanese Chinese?!

Let me guess, the excuse is going to be the same as for the Release Candidate version of CMake? It's an automated process and so 🤷